platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -1743,6 +1742,7 @@ public boolean onScale(ScaleGestureDetector detector) {
             if (uiSettings.isScrollGesturesEnabled() && !mQuickZoom && trackingSettings.isLocationTrackingDisabled()) {
                 // around gesture
                 mNativeMapView.scaleBy(detector.getScaleFactor(), detector.getFocusX() / mScreenDensity, detector.getFocusY() / mScreenDensity);
+                trackGestureEvent(MapboxEvent.GESTURE_QUICK_ZOOM, detector.getFocusX(), detector.getFocusY());
```

Comment:
@tobrun Yeah, this is a weird one.  When I placed log statements in between the first if and else as well as the if / else in the else statement (who's on first?) and did the single finger double tap and drag this was the code the fired.  I'm thinking there may be an issue with how `mTwoTap` gets set.